Spigot-API-Patches/0231-added-PlayerChunkReceiveEvent.patch

```diff
@@ -0,0 +1,53 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: ysl3000 <yannicklamprecht@live.de>
+Date: Mon, 5 Oct 2020 21:24:45 +0200
+Subject: [PATCH] added PlayerChunkReceiveEvent
+
+
+diff --git a/src/main/java/io/papermc/paper/event/packet/PlayerChunkReceiveEvent.java b/src/main/java/io/papermc/paper/event/packet/PlayerChunkReceiveEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..d6966e5b613c3a8c7dd3b4595d711ac24c403695
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/event/packet/PlayerChunkReceiveEvent.java
+@@ -0,0 +1,41 @@
++package io.papermc.paper.event.packet;
++
++import org.bukkit.Chunk;
++import org.bukkit.entity.Player;
++import org.bukkit.event.HandlerList;
++import org.bukkit.event.world.ChunkEvent;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * Is called before a {@link org.bukkit.entity.Player} receives a {@link org.bukkit.Chunk}
++ * used for e.g. Spawning a fake entity when the player receives a chunk
++ *
++ * Should only be used for packet/clientside related stuff.
++ * Not intended for modifying server side.
++ */
++public class PlayerChunkReceiveEvent extends ChunkEvent {
```

Comment:
Should also have a PlayerChunkUnloadEvent to complement this. So maybe name this PlayerChunkLoadEvent and PlayerChunkUnloadEvent

Reply:
Applied.